doc(release): Release notes for web 23.10.12, gorgone 23.10.7 (#3321)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-23.10/releases/centreon-os.mdx

## Centreon Web

### 23.10.12

Release date: `May 29, 2024`

<details>
  <summary>Bug fixes</summary>

- [API] Fixed an issue affecting downtimes on services linked to a service group via a service template.
- [API] Fixed an issue with is_password when using Service and Servite Template POST endpoint.
- [API] Fixed access control for some v2 API endpoints.
- [Auto Discovery] Fixed rights of temporary files created during configuration generation.
- [Configuration] Fixed an issue where the Service Categories configuration page could appear blank.
- [Packaging] Fixed symbolic link between /usr/lib/nagios/plugins and /usr/lib64/nagios/plugins.

> - [Resource Status] Fixed access to view report from resource details panel.

Reviewer: ```suggestion
- [Resource Status] Fixed access to "View report" icon in the resource details panel.
```

> - [Resource Status] Fixed the selection of disabled poller in filters.

Reviewer: ```suggestion
- [Resource Status] Disabled pollers can no longer be selected in filters.
```

> - [UX] Fixed the possibility to close the top counter pop-up when user clicks away.

Reviewer: ```suggestion
- [UX] Fixed an issue where the top counter pop-up was not closed when users clicked outside of it.
```

</details>

<details>
  <summary>Security fixes</summary>

- [Security] Fixed several SQLi issues.

> - [Security] Fixed XSS in host configuration.

Reviewer: ```suggestion
- [Security] Fixed XSS in a configuration page.
```
